app/src/views/Posts.vue

```diff
@@ -137,6 +137,12 @@ export default Vue.extend({
       // recalc header space after search tags change
       this.$nextTick(() => this.calcHeaderSpace());
     },
+    title: {
+      immediate: true,
+      handler() {
+        document.title = "Angebotssuche - Einander Helfen";
```

Comment:
I think something like "Ergebnisse" or "Angebote" or "Suchergebnisse" would be more fitting, because the posts resulted by the search are more important for this site than the search functionality.